### PR TITLE
Add documentation examples for SSE.

### PR DESCRIPTION
**Pull request by janisz**
Fixes: #5200
Releates: #4637

### CODE REVIEW

**Comment by jasongilanfarr** [comment]
Issues would do too. We need to make sure this lands on master then all relevant releases.

**Comment by janisz** [comment]
@jasongilanfarr I can create PR for 1.3 and 1.4 once this will be approved and merged. This should be easily to apply since this files hasn't been touched.

@tobilg @sascala Can you take a look at this?

File: docs/docs/event-bus.md [hunk 1]
```diff
@@ -29,6 +29,34 @@ Other subscribers are easy to add. See the code in
 [marathon/event/http](https://github.com/mesosphere/marathon/tree/master/src/main/scala/mesosphere/marathon/event/http)
 for guidance.
 
+## Subscription to Events via event stream
+
+This functionality is always enabled. Marathon implements
+[Server-sent events](https://en.wikipedia.org/wiki/Server-sent_events) standard.
```
Comment (sascala) [inline docs/docs/event-bus.md]: Server-Sent-Events (SSE)

File: docs/docs/event-bus.md [hunk 2]
```diff
@@ -29,6 +29,34 @@ Other subscribers are easy to add. See the code in
...
+Events are published on `/v2/events` endpoint. Any SSE compatible client
+could be used to subscribe. Example subscription using `curl`:
```
Comment (sascala) [inline docs/docs/event-bus.md]: "Any SSE-compatible client can subscribe."

File: docs/docs/event-bus.md [hunk 3]
```diff
@@ -29,6 +29,34 @@ Other subscribers are easy to add. See the code in
...
+
+```bash
+$ curl -H "Accept: text/event-stream"  <MARATHON_HOST>:<MARATHON_PORT>/v2/events
+
+event: event_stream_attached
+data: {"remoteAddress":"127.0.0.1","eventType":"event_stream_attached","timestamp":"2017-02-18T19:12:00.102Z"}
+```
+
+### Filtering event stream
```
Comment (sascala) [inline docs/docs/event-bus.md]: Filtering the Event Stream

File: docs/docs/event-bus.md [hunk 4]
```diff
@@ -29,6 +29,34 @@ Other subscribers are easy to add. See the code in
...
+
+From version [1.3.7](https://github.com/mesosphere/marathon/releases/tag/v1.3.7)
```
Comment (sascala) [inline docs/docs/event-bus.md]: Suggest: "Starting from version [1.3.7], Marathon supports filtering the event stream by event type. To filter by event type, specify a value for the `event_type` parameter in your `/v2/events` request."

Where can a user find the available event types?

File: docs/docs/event-bus.md [hunk 5]
```diff
@@ -29,6 +29,34 @@ Other subscribers are easy to add. See the code in
...
+Marathon does support filtering event stream by event type.
+This could be done by adding interesting event type as value for `event_type`
+parameter to `/v2/events` request.
+
+Following example will subscribe only events about new client attaching or
```
Comment (sascala) [inline docs/docs/event-bus.md]: The following example only subscribes to events involving a new client attaching to or detaching from the event stream.

File: docs/docs/event-bus.md (hunk 4 above)
Comment (janisz) [inline docs/docs/event-bus.md]: Currently user can find available event types only in code. There is dedicated issue for this https://github.com/mesosphere/marathon/issues/4672

**Comment by janisz** [comment]
@sascala Thanks for review. I fixed issues and updated PR.

**Comment by aquamatthias** [comment]
ok to test

**Comment by janisz** [comment]
@sascala @aquamatthias  Can we merge this?

**Comment by sascala** [comment]
@janisz merged

**Pull request merged**